.github/workflows/deploy-gh.yml

```diff
@@ -74,13 +74,6 @@ jobs:
           path: node_modules
           key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package.json') }}
 
-      - name: Cache dist
-        uses: actions/cache@v4
-        id: cache-dist
-        with:
-          path: build
-          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ github.sha }}
-
       - name: Cache runs
```

Comment:
@danielfdsilva I'm still using the cached runs here. Maybe we can first see if using the build artifact solves the issue before trying to alter this?